designs/wahid/src/frontlining.js

```diff
@@ -23,8 +22,21 @@ export default (part) => {
     paths.seam.curve(points.splitDartHemRightCp2, points.splitHemCp1, points.hem)
   } else paths.seam.line(points.hem)
   paths.seam.close()
-
   if (complete) {
+    //Grainline
+    let grainlineVariableLength = points.armhole.dist(points.hem)
+    let grainlineVariableShift = points.dartHipRightCpTop.dist(points.hipsCp2) * 0.1
+    points.grainlineFromLining = new Point(points.dartEnd.x, points.dartEnd.y)
+      .shift(90, grainlineVariableLength)
+      .shift(0, grainlineVariableShift)
+    points.grainlineToLining = new Point(points.dartEnd.x, points.dartEnd.y).shift(
+      0,
+      grainlineVariableShift
+    )
+    macro('grainline', {
```

Comment:
Here too, can't we just:
```js
macro('grainline', {
  from: points.flbTop,
  to: new Point(points.flbTop.x, points.cfHem.y)
})
```

(note that you'll need to add `Point` to the shorthand call to do this)